framework/components/ecl-carousels/ecl-carousels.config.js

```diff
@@ -0,0 +1,95 @@
+module.exports = {
+  title: 'Carousels',
+  label: 'Carousels',
+  status: 'ready',
+  tags: ['molecule'],
+  context: {
+    _demo: {
+      scripts: `
+        document.addEventListener('DOMContentLoaded', function () {
+            ECL.carousels();
+        });
+      `,
+    },
+    extra_attributes: [
+      {
+        name: 'aria-labelledby',
+        value: 'ecl-carousel__heading',
+      },
+      {
+        name: 'id',
+        value: 'ecl-carousel',
+      },
+    ],
+    heading_title: 'Example gallery carousel',
+    heading_attributes: [
+      {
+        name: 'id',
+        value: 'ecl-carousel__heading',
+      },
+      {
+        name: 'class',
+        value: 'ecl-heading--3 ecl-sr-only',
```

Comment:
Thanks, I changed it. Since footer, every element should get our namespaces correctly, even if sometimes it's not really necessary, but it's good for consistency! :)